src/clusterfuzz/_internal/bot/tasks/update_task.py

```diff
@@ -39,10 +39,18 @@
 from clusterfuzz._internal.system import process_handler
 from clusterfuzz._internal.system import shell
 
+CLUSTERFUZZ_RELEASE = os.getenv('CLUSTERFUZZ_RELEASE')
+if not CLUSTERFUZZ_RELEASE:
+  CLUSTERFUZZ_RELEASE = 'prod'
+
 TESTS_LAST_UPDATE_KEY = 'tests_last_update'
 TESTS_UPDATE_INTERVAL_DAYS = 1
 
-MANIFEST_FILENAME = 'clusterfuzz-source.manifest.3'
+REMOTE_MANIFEST_FILENAME = 'clusterfuzz-source.manifest'
```

Comment:
It feels like there's a lot of repetition everywhere with this logic to compute the suffixes. Can we instead just create helpers for this that can be used everywhere?

e.g. something along the lines of 

```python
def get_source_zip_suffix():
  ...

def get_source_zip_name():
  ...
```

We can remove some of these constants if necessary and just make them dynamically computed based on reading from env.

Reply:
Moved this to clusterfuzz._internal.base.utils so butler package/deploy and update_task can share this code